src/templates/_page_list.php

```diff
@@ -4,10 +4,12 @@
 
 ?>
 
-<div id="spw-page-list">
-  <ul>
+<div id="spw-page-list" class="mole-sidebar-object-div">
```

Comment:
Could you please update your class names to follow the `spw-` naming convention?